WalletWasabi.Fluent/Helpers/CoinPocketHelper.cs

```diff
@@ -14,11 +14,16 @@ public static class CoinPocketHelper
 	public static readonly SmartLabel PrivateFundsText = new("Private Funds");
 	public static readonly SmartLabel SemiPrivateFundsText = new("Semi-private Funds");
 
-	public static IEnumerable<(SmartLabel SmartLabel, ICoinsView Coins)> GetPockets(this ICoinsView allCoins, int privateAnonSetThreshold)
+	public static IEnumerable<(SmartLabel SmartLabel, ICoinsView Coins)> GetPockets(this ICoinsView allCoins, int privateAnonSetThreshold, bool onlyAvailableCoins = false)
 	{
 		List<(SmartLabel SmartLabel, ICoinsView Coins)> pockets = new();
 		var clusters = new Dictionary<SmartLabel, List<SmartCoin>>();
 
+		if (onlyAvailableCoins)
+		{
+			allCoins = allCoins.Available();
```

Comment:
What if you just pass `allCoins` from outside in a way that it does not contain coinjoining coins.

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/9706#discussion_r1047007606